get_http.go

```diff
@@ -117,13 +118,54 @@ func (g *HttpGetter) GetFile(dst string, u *url.URL) error {
 		g.Client = httpClient
 	}
 
-	resp, err := g.Client.Get(u.String())
+	// check to see whether user has specified a range of bytes to download.
+	// if user has, but the range is invalid, fall back to downloading the
+	// whole file
+	isPartialDownload := false
```

Comment:
this is a trivial comment, but it might make sense to move this closer down to where it's being used.

Reply:
I guess I could move it down to ~ L134 if you think it'll make it read clearer...